HEN_HOUSE/user_codes/dosxyznrc/dosxyznrc.mortran

```diff
@@ -1518,13 +1519,14 @@ IF(NMED = 0)[ "Input data from CT data file as processed by the code ctcreate"
       ' z range : ',F12.5,' - ',F12.5,' cm');
 
     "input the media numbers"
+    encoding = '0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz';
     DO k=1,KMAX[
        DO j=1,JMAX[
           "Read in the characters that correspond to medium numbers"
           read($CTUnitNumber,:medformat:) (medChar(i),i=1,IMAX);
           "Convert from ASCII to a number"
           DO i=1,IMAX[
-            med($IR(i,j,k)) = mod((iachar(medChar(i)) + 47), 95);
+            med($IR(i,j,k)) = index(encoding, medChar(i)) - 1;
```

Comment:
I didn't test this line, @ftessier did you?

Reply:
Not yet!

Reply:
@rtownson do you have a ctcreate'd phantom around on which to test this conversion?

Reply:
I don't know about your compilers, but I ran into a compilation error complaining that encoding had been defined before all variables had been declared.  I got rid of this by moving the "encoding = $ENCODING;" statement to immediately before "call egs_init;"

Reply:
Strange; it compiled fine here. We should be able to define a variable locally in the write_phantom subroutine, no?

Reply:
Strange, it compiled fine here. We should be able to define it as a local variable in the write_phantom subroutine, no? What is going on?

Reply:
Ah ok, I see, I was looking at the _other_ one in write_phantom. Yes, ok, I see what you mean, will fix right away.

Reply:
Fixed, but leaving this conversation unresolved because this still requires a round of testing.

Reply:
Resolved with testing by @blakewalters.